packages/editor/src/components/provider/index.js

```diff
@@ -67,55 +66,40 @@ class EditorProvider extends Component {
 		} );
 	}
 
-	componentDidUpdate( prevProps ) {
-		if ( this.props.settings !== prevProps.settings ) {
-			this.props.updateEditorSettings( this.props.settings );
-		}
-	}
-
 	render() {
-		const {
-			children,
-		} = this.props;
-
-		const providers = [
-			// Slot / Fill provider:
-			//
-			//  - context.getSlot
-			//  - context.registerSlot
-			//  - context.unregisterSlot
-			[
-				SlotFillProvider,
-			],
-
-			// DropZone provider:
-			[
-				DropZoneProvider,
-			],
-		];
-
-		const createEditorElement = flow(
-			providers.map( ( [ Provider, props ] ) => (
-				( arg ) => createElement( Provider, props, arg )
-			) )
+		const { children, settings, blocks, updateEditorBlocks } = this.props;
+
+		return (
+			<BlockEditorProvider
+				value={ blocks }
+				onChange={ updateEditorBlocks }
+				settings={ settings }
+			>
```

Comment:
Here's how you'd use a `BlockEditorProvider` component, inside it you'd probably have to use a `BlockInserter`, `BlockList` and `BlockInspector` components to build your UI.

Reply:
> Here's how you'd use a `BlockEditorProvider` component, inside it you'd probably have to use a `BlockInserter`, `BlockList` and `BlockInspector` components to build your UI.

We should make a note to document this somewhere once the components are in their correct locations.

Reply:
Agreed, once the block editor module is more polished (with component and refactoring to the store used), I'll add some documentation and I'll also plan to add a built-in static playground version of gutenberg using this module inside the repository.